Automattic/jetpack
# Moves autoloader code into a separate file and renames class/file maps.

This change moves the code that is setting up the autoloader into a separate file for easier development.




#### Changes proposed in this Pull Request:
* Moves autoloader setup into a separate file.
* Renames class and file map files to distinguish them from the old autoload files.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* This is part of the DNA project effort.

#### Testing instructions:
* Make sure `composer install` still works.
* Make sure the plugin works as expected, nothing should change with on this PR.

#### Proposed changelog entry for your changes:
* N/A
